onyo/commands/set.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+
+import logging
+import os
+import sys
+import yaml
+import glob
+from ruamel.yaml import YAML
+
+from git import Repo
+
+from onyo.utils import (
+    run_cmd,
+    build_git_add_cmd,
+    validate_rule_for_file
+)
+
+logging.basicConfig()
+logger = logging.getLogger('onyo')
+
+
+def build_commit_cmd(files_to_change, onyo_root):
+    return ["git -C " + onyo_root + " commit -m", "set values.\n\n" +
+            "\n".join(files_to_change)]
+
+
+def read_asset(file, onyo_root):
+    yaml = YAML(typ='safe')
+    asset = {}
+    with open(os.path.join(onyo_root, file), "r") as stream:
+        try:
+            asset = yaml.load(stream)
+        except yaml.YAMLError as e:
+            print(e)
+    if asset is None:
+        return {}
+    return asset
+
+
+def write_asset(data, file, onyo_root):
+    with open(os.path.join(onyo_root, file), "w") as stream:
+        yaml.dump(data, stream)
+
+
+def set_value(file, git_filepath, keys, onyo_root):
+    asset = read_asset(git_filepath, onyo_root)
+    for key in keys.items():
+        try:
+            asset[key[0]] = key[1]
+        # when in YAML asset just a line with key like "RAM" without value:
```

Comment:
Missing "is" (I think)

Reply:
Updated to "when a YAML asset just has a line with a key like "RAM" without value:"